metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/operations/elastic/OperationsController.java

```diff
@@ -0,0 +1,83 @@
+package io.datahubproject.openapi.operations.elastic;
+
+import com.datahub.authentication.Authentication;
+import com.datahub.authentication.AuthenticationContext;
+import com.datahub.authorization.AuthUtil;
+import com.datahub.authorization.AuthorizerChain;
+import com.datahub.authorization.ConjunctivePrivilegeGroup;
+import com.datahub.authorization.DisjunctivePrivilegeGroup;
+import io.datahubproject.openapi.util.ElasticsearchUtils;
+import com.google.common.collect.ImmutableList;
+import com.linkedin.metadata.authorization.PoliciesConfig;
+import com.linkedin.metadata.systemmetadata.SystemMetadataService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.elasticsearch.client.tasks.GetTaskResponse;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.propertyeditors.StringArrayPropertyEditor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.InitBinder;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+@RequestMapping("/elastic")
```

Comment:
Elastic is a bit generic, there are many things it could refer to. If this is only for ElasticSearch operations would prefer that to be reflected in the Controller name and have it fully specified in the request mapping "elasticSearch" or "elasticSearchOperations"